backend/app/api/ws/chat_namespace.py

```
@@ -1001,169 +976,56 @@ async def on_chat_cancel(self, sid: str, data: dict) -> dict:
-                    from app.services.chat.trigger import (
-                        notify_group_members_task_updated,
-                    )
-
-                    await notify_group_members_task_updated(
-                        db=db, task=task, sender_user_id=user_id
-                    )
+            # Note: Database updates and WebSocket events are handled by StatusUpdatingEmitter
+            # when the executor sends response.incomplete event through the dispatcher chain.
+            # This ensures unified status update logic across all execution modes.
+            logger.info(
+                f"[WS] chat:cancel Cancel request sent, status updates will be handled by StatusUpdatingEmitter"
```

Ruff (0.15.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)